<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkSimPrefWidget</class>
 <widget class="QWidget" name="WkSimPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="startPausedCheckBox">
     <property name="text">
      <string>Start Simulations Paused</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="draggablePlatformsCheckBox">
     <property name="text">
      <string>Allow Dragging of Platforms to Set New Position</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="simClockRateLabel">
       <property name="text">
        <string>Simulation Clock Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="simClockRateComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="disGroupBox">
     <property name="title">
      <string>DIS Settings</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>This block will override the DIS settings defined in a scenario to use the settings specified in this block.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QSpinBox" name="portSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Exercise Id</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Multicast IP Address</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="exerciseIdSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Site Id</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="UtQtMulticastIpv4LineEdit" name="ipAddressLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="siteIdSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="UtQtIpv4LineEdit" name="netIdLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Application Id</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="applicationIdSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Interface</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="multicastHelpPushButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="interfaceHelpPushButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="deferredStartGroupBox">
        <property name="title">
         <string>Deferred Start Time</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="deferredStartTimeLabel">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="UtQtTimeValueEdit" name="deferredStartTimeLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Note: The simulation must be restarted for these changes to take effect.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtIpv4LineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtNetwork.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtMulticastIpv4LineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtNetwork.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtTimeValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startPausedCheckBox</tabstop>
  <tabstop>disGroupBox</tabstop>
  <tabstop>ipAddressLineEdit</tabstop>
  <tabstop>multicastHelpPushButton</tabstop>
  <tabstop>netIdLineEdit</tabstop>
  <tabstop>interfaceHelpPushButton</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>siteIdSpinBox</tabstop>
  <tabstop>applicationIdSpinBox</tabstop>
  <tabstop>exerciseIdSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
